x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -123,21 +124,17 @@ export function SloListItem({
     setIsActionsPopoverOpen(false);
   };
 
+  const handleDeleteConfirm = () => {
+    setDeleteConfirmationModalOpen(false);
+    onConfirmDelete(slo);
+  };
+
   const handleDeleteCancel = () => {
     setDeleteConfirmationModalOpen(false);
   };
 
   return (
-    <EuiPanel
-      data-test-subj="sloItem"
-      color={isDeletingSlo ? 'subdued' : undefined}
-      hasBorder
-      hasShadow={false}
-      style={{
```

Comment:
As we optimistically update, this style was never seen anymore. Might as well remove it